.github/workflows/welcome-bot.yml

```
@@ -1,19 +1,30 @@
+
 name: Welcome First-Time Contributor
 
+name: Welcome Bot
```

actionlint (1.7.12)

[error] key "name" is duplicated in "workflow" section. previously defined at line:2,col:1

(syntax-check)